examples/helpers/tiled.rs

```diff
@@ -218,32 +306,36 @@ pub fn process_loaded_maps(
                             }
                         }
 
+                        let texture = if tile_images.len() == 1 {
+                            TilemapTexture::Single(tile_images[0].clone_weak())
+                        } else {
+                            #[cfg(feature = "atlas")]
+                            panic!("Cannot use Tiled maps with a tileset that uses a collection of images due to incompatibility with the 'atlas' feature.");
```

Comment:
This should not really panic [because the tileset should be skipped](https://github.com/StarArawn/bevy_ecs_tilemap/pull/381/files#diff-45eb609402711b3323b6becf33654bed7e7e2dcf289f2d0f312a9f13565846e9R195-R198) earlier in the loop. However I wasn't sure how to satisfy the compiler without an even more extreme solution, such as generating a placeholder `TilemapTexture::Single()` here.

Reply:
I had an idea for cleaning up this and related code: https://github.com/rparrett/bevy_ecs_tilemap/commit/b9e707c360969163ad8108559ec7b926b91ed2b7

Reply:
Unfortunately, this turned out to be a bad idea, as it also can't compile with the atlas feature.

Reply:
Fixed that up in a new commit here: https://github.com/rparrett/bevy_ecs_tilemap/commit/b794f2d17fa7cb24495d3416dd995eaf1176782f which seems to be behaving.

This seems to be behaving, apart from some warnings that should probably be cleaned up when compiled with `atlas`

Reply:
That looks good to me. Works in my game. I like the idea of creating the `Single` and `Vector` structs earlier in the loading process. I had tried to do that but ran into some ownership issues and dialed it back for time.